libcxx/test/std/utilities/utility/mem.res/mem.poly.allocator.class/mem.poly.allocator.class.general/equality.pass.cpp

```diff
@@ -0,0 +1,41 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// UNSUPPORTED: c++03, c++11, c++14
+// UNSUPPORTED: availability-pmr-missing
```

Comment:
This comment still belongs here I think:

```
// TODO: Change to XFAIL once https://github.com/llvm/llvm-project/issues/40340 is fixed
```

Per my comment in https://github.com/llvm/llvm-project/pull/100884/files

Reply:
Fixed in [2fe08ed](https://github.com/llvm/llvm-project/commit/2fe08ed35a968dcfd797cbd31ae56bb0b1447faf)